contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,28 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+
+CREATE OR REPLACE VIEW information_schema_tsql.domain_constraints
+AS SELECT sys.db_name()::information_schema.sql_identifier AS constraint_catalog,
+    rs.nspname::information_schema.sql_identifier AS constraint_schema,
+    con.conname::information_schema.sql_identifier AS constraint_name,
```

Comment:
Use sys datatypes (eg. sys.nvarchar(128)), refer to Microsoft documentation for this